snapcraft.yaml

```diff
@@ -1666,6 +1666,16 @@ parts:
       rm -f usr/lib/*.la
       rm -f usr/lib/*/*.la
 
+      # cleanup broken links; all these are from libc6-dev, libcrypt-dev and libpciaccess-dev
```

Comment:
Although they should indeed be removed, the AR files are not broken links, so could you just make this more accurate in this comment?